libs/blocks/global-navigation/global-navigation.js

```diff
@@ -526,26 +527,32 @@ class Gnav {
 
     const onAnalyticsEvent = (data) => {
       if (!data) return;
-
       const getInteraction = () => {
-        const {
-          event: { type, subtype } = {},
-          source: { name } = {},
-          content: { name: contentName } = {},
-        } = data;
-
-        switch (`${name}|${type}|${subtype}|${contentName || ''}`) {
-          case 'profile|click|sign-in|':
+        const { workflow, type, subtype } = data.event || data;
```

Comment:
@robert-bogos - question question on this: is there any use for `data` if the `data.event` object is not available? Will UNC use `data.workflow` directly ?

Reply:
I made some changes here, so there's no `data.event || data` anymore 👍
`data` was used for UNC, as it had no `data.event`